pilot/pkg/kube/inject/inject.go

```diff
@@ -345,6 +347,16 @@ func ValidateExcludeInboundPorts(ports string) error {
 	return validatePortList("excludeInboundPorts", ports)
 }
 
+// ValidatePolicyCheck validates policy check override
+func ValidatePolicyCheck(value string) error {
+	switch value {
+	case mixer.PolicyCheckEnable, mixer.PolicyCheckDisable, mixer.PolicyCheckFailOpen:
```

Comment:
nit: i'd be inclined to leave a comment or a return nil or something here to indicate that these are the valid values for readability.